sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -52,9 +52,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>clearFaultState</EFDB_Name>
```

Comment:
called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."